MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -47,41 +47,58 @@ internal class RealMiniApp {
     /// will be passed on to Downloader class (which will check whether the mini app is downloaded already if not, it will download)
     /// - Parameters:
     ///   - appInfo: Miniapp Info object
+    ///   - manifest: optional Mini App manifest containing permissions and other metadata
     ///   - queryParams: Optional Query parameters that the host app would like to share while creating a mini app
     ///   - completionHandler: Completion Handler that needed to pass back the MiniAppDisplayProtocol
     ///   - messageInterface: Miniapp communication protocol object.
     ///   - adsDisplayer: a delegate that will handle Miniapp ads requests
-    func createMiniApp(appInfo: MiniAppInfo, queryParams: String? = nil, completionHandler: @escaping (Result<MiniAppDisplayDelegate, Error>) -> Void, messageInterface: MiniAppMessageDelegate? = nil, adsDisplayer: MiniAppAdDisplayer? = nil) {
+    func createMiniApp(appInfo: MiniAppInfo, manifest: MiniAppManifest? = nil, queryParams: String? = nil, completionHandler: @escaping (Result<MiniAppDisplayDelegate, Error>) -> Void, messageInterface: MiniAppMessageDelegate? = nil, adsDisplayer: MiniAppAdDisplayer? = nil) {
```

Comment:
I don't understand the need for passing the `manifest` object from the Host app. Irrespective of what manifest sent from the Host app, SDK will be checking from platform/cache right? Then we won't be needing this one.

Or my understanding is wrong here ?

Reply:
we don't need anymore, I think this was my first implementation, and this did not appear during my refactoring